ARExperiment/ARExperiment/Helpers.swift

```diff
@@ -0,0 +1,146 @@
+import Foundation
+import UIKit
+import CoreML
+import Accelerate
+
+// The labels for the 20 classes.
+let labels = [
+  "aeroplane", "bicycle", "bird", "boat", "bottle", "bus", "car", "cat",
+  "chair", "cow", "diningtable", "dog", "horse", "motorbike", "person",
+  "pottedplant", "sheep", "sofa", "train", "tvmonitor"
+]
+
+let anchors: [Float] = [1.08, 1.19, 3.42, 4.41, 6.63, 11.38, 9.42, 5.11, 16.62, 10.52]
+
+/**
+  Removes bounding boxes that overlap too much with other boxes that have
+  a higher score.
+
+  Based on code from https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/kernels/non_max_suppression_op.cc
+
+  - Parameters:
+    - boxes: an array of bounding boxes and their scores
+    - limit: the maximum number of boxes that will be selected
+    - threshold: used to decide whether boxes overlap too much
+*/
+func nonMaxSuppression(boxes: [YOLO.Prediction], limit: Int, threshold: Float) -> [YOLO.Prediction] {
+
+  // Do an argsort on the confidence scores, from high to low.
```

Comment:
When you have to write comments as this one, is better to extract this logic into a well named function

Reply:
This was a file created by the person who wrote the model and the Yolo class that is why I did not change it.